VectorizedAlgorithms/ElonAbernathy_Project.cs

```diff
@@ -24,9 +22,7 @@ public class ElonAbernathy_Project
         public LineSegment[] Segments { get; private set; }
         private VecPoint[] vecPoints;
         private VecSegment[] vecSegments;
-
```

Comment:
The empty lines are here for readability, friend. Please refrain from removing them.